src/CSnakes/Reflection/MethodReflection.cs

```diff
@@ -202,4 +215,121 @@ private static ReturnStatementSyntax ProcessMethodWithReturnType(TypeSyntax retu
                     .WithArgumentList(ArgumentList()));
         return returnExpression;
     }
+
+    private static InvocationExpressionSyntax GenerateParamsCall(List<(PythonFunctionParameter pythonParameter, ParameterSyntax cSharpParameter)> parameterList)
+    {
+        List<ArgumentSyntax> pythonFunctionCallArguments = parameterList.Select((a) => Argument(IdentifierName($"{a.cSharpParameter.Identifier}_pyObject"))).ToList();

```

Comment:
Just a note, unless you are wanting to call List operations (like Add/Remove/etc.) there's no need to turn this into a List, it can be left as `IEnumerable<>` and passed along. This will result in lazy evaluation in the `SeparatedList` method call.